demos/instance_segmentation_demo/python/instance_segmentation_demo/model_utils.py

```diff
@@ -14,9 +14,14 @@
  limitations under the License.
 """
 
+import sys
+from pathlib import Path
+sys.path.append(str(Path(__file__).resolve().parents[3] / 'common/python/models'))
+
```

Comment:
This shouldn't be needed if the main script of the demo already modifies `sys.path` (which it does).

Reply:
Updated.